### PR TITLE
Get history of git project

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #147 
Add a section to explain how to use the `git log` command to get an idea of the history of a project.
describe the option
- git log --oneline
- git log --graph
- git log --merges --first-parent master

### CODE REVIEW

File: ProjectScopingTips.md [hunk 1]
```diff
@@ -47,8 +47,36 @@ You may tell the client that your call will address the following:
...
+![image](https://user-images.githubusercontent.com/6057298/53473424-41673680-3a62-11e9-8bd1-808c88d418a9.png)
+
+  - **`git log --oneline --merges --first-parent master`** 
+  will only display the merged commits into master <br />
+  (_i.e. merged pull requests_)
+![image](https://user-images.githubusercontent.com/6057298/53473760-2fd25e80-3a63-11e9-8751-d9f957d4b15a.png)
+
+  - **`git log --oneline --merges --first-parent master | wc -l`** 
+  Finally using the pipe command with the above history log and `wc -l` (count number of line)
+we get the number of pull requests merged into master
```
Comment (nelsonic) [inline ProjectScopingTips.md]: @SimonLab are you considering adding a screenshot for this final command (_with pipe_) ?

Comment (SimonLab) [reply ProjectScopingTips.md]: I was wondering if it's useful as the command return only a number, at the moment 51 (if you check on Github you'll see that the number is 52 but one of the PR hasn't been merged and was just close)
I could add something like this:
```bash
> 52
```
?

Comment (nelsonic) [reply ProjectScopingTips.md]: Even if it's just a two line screenshot, it gives people an idea of what to _expect_ 😉

Comment (SimonLab) [reply ProjectScopingTips.md]: better?
image

Comment (nelsonic) [reply ProjectScopingTips.md]: I think it adds value. Thanks Simon! 😊

**Pull request merged**